java/junit4/src/test/java/com/saucelabs/saucebindings/junit4/examples/DataCenterExample.java

```diff
@@ -0,0 +1,37 @@
+package com.saucelabs.saucebindings.junit4.examples;
+
+import com.saucelabs.saucebindings.DataCenter;
+import com.saucelabs.saucebindings.SauceSession;
+import com.saucelabs.saucebindings.junit4.SauceBindingsWatcher;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+
+public class DataCenterExample {
+  private SauceSession session;
+  private WebDriver driver;
+
+  // 1. Pass in desired Datacenter to the SauceBindingsWatcher rule
+  @Rule
+  public SauceBindingsWatcher sauceWatcher =
+      new SauceBindingsWatcher.Builder().withDataCenter(DataCenter.EU_CENTRAL).build();
```

Comment:
Alternately we could/should do:
```
SauceBindingsWatcher.builder().withDataCenter(DataCenter.EU_CENTRAL).build()
```
The point is to get away from creating 6 different constructors in SauceBindingsWatcher
```
public SauceBindingsWatcher(SauceOptions options, DataCenter, dataCenter) {}
public SauceBindingsWatcher(Capabilities caps, DataCenter, dataCenter) {}
public SauceBindingsWatcher() {}
public SauceBindingsWatcher(SauceOptions options) {}
public SauceBindingsWatcher(Capabilities caps) {}
public SauceBindingsWatcher(DataCenter dataCenter) {}
```